pkg/pushsync/pushsync.go

```diff
@@ -314,18 +248,22 @@ func (ps *PushSync) PushChunkToClosest(ctx context.Context, ch swarm.Chunk) (r *
 			}
 		}
 
-		receipt, err := ps.receiveReceipt(ctx, r)
-		if err != nil {
+		var receipt pb.Receipt
+		cctx, cancel := context.WithTimeout(ctx, timeToWaitForReceipt)
+		defer cancel()
+		if err := r.ReadMsgWithContext(cctx, &receipt); err != nil {
 			ps.metrics.TotalErrors.Inc()
 			_ = streamer.Reset()
 			lastErr = fmt.Errorf("chunk %s receive receipt from peer %s: %w", ch.Address().String(), peer.String(), err)
-			logger.Debugf("pushsync-push: %v", lastErr)
+			logger.Errorf("pushsync error: %v", lastErr)
```

Comment:
Error word in message is redundant, it already is an error log line. I am suggesting `pushsync: read message: %v`